open_bus_stride_api/routers/common.py

```diff
@@ -166,6 +181,15 @@ def param_limit(max_limit=100):
     return fastapi.Query(None, description=f'Limit the number of results up to {max_limit}. If not specified will limit to {max_limit} results. Use the offset param to get more results.')
 
 
+def doc_param(what_singular: str, filter_type: str, description: str = "", example: str = "", default: str = None):
```

Comment:
@OriHoch LMK if this api is fine by you, and I'll refactor the rest of the usages to use it.

Reply:
yes, it's great, we can use it